k3k-kubelet/config.go

```diff
@@ -54,6 +55,9 @@ func (c *config) unmarshalYAML(data []byte) error {
 	if c.ServerIP == "" {
 		c.ServerIP = conf.ServerIP
 	}
+	if c.Version == "" {
+		c.Version = conf.Version
+	}
```

Comment:
I would use the new `cmp.Or` func to simplify this very long chain of ifs:

```suggestion
	c.Version = cmp.Or(c.Version, conf.Version)
```

Reply:
I am actually trying to think of a different way altogether to simplify this function, it just piled up with a lot of options to virtual kubelet